### PR TITLE
Add API service to demo deploys

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
https://ci.us-west.moz.works/blue/organizations/jenkins/mdn_multibranch_pipeline/detail/api/3/pipeline/

File: Makefile [hunk 1]
```diff
@@ -132,6 +132,10 @@ k8s-migrate:
 	$(shell kubectl --namespace ${DEIS_APP} get pods | grep ${DEIS_APP}-cmd | awk '{print $$1}') \
 	python manage.py migrate
 
+render-k8s-templates:
+	k8s/api-svc.yaml.template.sh > k8s/api-svc.yaml
+	rm k8s/api-svc.yaml.template.sh
```
Comment (metadave) [inline Makefile]: any reason for removing this script after rendering yaml?

File: k8s/api-svc.yaml.template.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  namespace: ${APP}
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: ${APP}
+    heritage: deis
```
Comment (metadave) [inline k8s/api-svc.yaml.template.sh]: do we need `heritage` here?

**Comment by metadave** [comment]
lgtm 👍

File: k8s/api-svc.yaml.template.sh (hunk 2 above)
Comment (escattone) [inline k8s/api-svc.yaml.template.sh]: I don't think it's necessary, but perhaps safer in that we're more specific in targeting the "app: api" container created by Deis via the Procfile? Very thin reason though given that we don't have any pods labelled with "app: api" other than the one created via the Deis Procfile! :)

I'll leave it in for now.

File: Makefile (hunk 1 above)
Comment (escattone) [inline Makefile]: I'll try it without removing the script to see what happens.

Comment (escattone) [inline Makefile]: It didn't seem to make any difference whether or not it was removed. Maybe it's safer to keep the removal just so there's no possibility of confusing ``kubectl``? I'll leave it in for now.

File: k8s/api-svc.yaml.template.sh (hunk 2 above)
Comment (jgmize) [inline k8s/api-svc.yaml.template.sh]: Good catch, it's not necessary, just something I forgot to strip out when copying from another service.

File: Makefile (hunk 1 above)
Comment (jgmize) [inline Makefile]: I wasn't sure if kubectl would ignore unsupported filenames off the top of my head so I added the `rm` for the initial iteration with the plan of removing it later if it turned out not to be necessary. Thanks for testing @escattone; I'll go ahead and remove it now.

Comment (metadave) [inline Makefile]: cool, we can also render text and pass it to stdin of `cat foo.json | kubectl create -f -` if needed.

**Pull request merged**